### DISCUSSION TITLE
Gap of Header Component in fixed Position can't be resolved with CSS reset

### DISCUSSION THREAD

**Discussion by StefanieLoeser** [neuefische/questions#150] [Web] [answered]
Hallo zusammen, 
in meiner nextjs-App rutscht die Header-Komponente beim Styling mit einer fixed Position immer nach unten. In main wird eine Liste gerendert, die eine margin hat. Ich habe bereits unterschiedlichste Resets auf margin:0 versucht (GlobalStyles sind auch eingerichtet) und komme zu keinem Ergebnis. Leider weiß ich nicht, an welcher Stelle der Fehler passiert und entsprechend nicht welchen Code-Schnipsel es Sinn macht zu posten. Anbei daher auch der Link zu meinem ersten Pull Request.
Vielen Dank für eure Hilfe!
Stefanie

[**---> Link zum PR**](https://github.com/StefanieLoeser/capstone-project/pull/8)

**Code Snippets**:

![Bildschirmfoto 2022-09-20 um 12 24 38](https://user-images.githubusercontent.com/104068626/191234212-9c5f8283-5e2d-4cc9-aa2d-26550a7293a3.png)
![Bildschirmfoto 2022-09-20 um 12 25 16](https://user-images.githubusercontent.com/104068626/191234315-9271246c-e077-4c79-9482-9d27c7bb6414.png)
![Bildschirmfoto 2022-09-20 um 12 25 42](https://user-images.githubusercontent.com/104068626/191234390-ee0dd75d-3618-4e7d-abe8-cb61e195e26e.png)

**Comment by NikRoe** [comment, marked as answer]
Hey Stefanie,
it looks like your styled component `Head` is missing a property. By adding `top: 0;` as another CSS property your headline should behave as intended :) 

On another note, I would advise renaming the component to something other than `Head`, since Next.js provides us with an identically named component. This might lead to confusion.
For more information on the `Head` component from Next.js, see this [link](https://nextjs.org/docs/api-reference/next/head)
